### PR TITLE
Update Home Page

### PR DESCRIPTION
**Pull request by cacheung**
Update Home Page

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/pages/documentation/index.md [hunk 1]
```diff
@@ -7,63 +7,56 @@ contributors:
...
-
-<InlineAlert variant="info" slots="header, text"/>
+<InlineAlert variant="success" slots="header, text"/>
 
-The Swift versions of the iOS SDKs have been released! 
+Good to go for iOS 16 and Android 13
 
-For more information on the Swift versions, including available Swift extensions and reference information, please read the [current SDK versions document](#).<br/><br/>Additionally, the Swift iOS SDKs are open source. You can learn more about this move to Swift and open source in the [Adobe Tech Blog](https://medium.com/adobetech/adobe-experience-platform-mobile-sdks-move-to-swift-for-ios-6aa67b67b4d4).
+Latest versions of the Adobe Experience Platform SDK for Google Android and Apple iOS (Swift & Objective-C) are tested for compatibilit with release verisons of Apple iOS 16 and Google Android 13. See [Current SDK Versions](./current-sdk-versions.md) to find our latest SDK versions or [release notes](./release-notes/index.md) for information on changes and updates.
```
Comment (calebk1) [inline src/pages/documentation/index.md:16]: A couple of misspelled words here - should be "The latest versions of the Adobe Experience Platform SDK for Google Android and Apple iOS (Swift and Objective-C) are tested for compatibility with release versions of..."

**Review by calebk1 — COMMENTED** [review]
One comment!

File: src/pages/documentation/index.md (hunk 1 above)
Comment (cacheung) [reply src/pages/documentation/index.md]: updated.

File: src/pages/documentation/index.md [hunk 2]
```diff
@@ -7,63 +7,56 @@ contributors:
...
+- [React Native (AEP 1.x)](./current-sdk-versions.md#react-native-aep-1x)
+- [React Native (ACP 2.x)](./current-sdk-versions.md#react-native-acp-2x)
+- [Flutter (AEP 1.x)](./current-sdk-versions.md#flutter-aep-1x)
+- [Flutter (ACP 2.x)](./current-sdk-versions.md#flutter-acp-2x)
+- [Cordova](./current-sdk-versions.md#cordova)
+- [WeChat Mini Programs](./platform-minis)
+- [Xamarin](./current-sdk-versions.md#xamarin)
 
-## Get help
+<InlineAlert variant="info" slots="header, text"/>
```
Comment (emdobrin) [inline src/pages/documentation/index.md:47]: we can use "warning" here

File: src/pages/documentation/index.md [hunk 3]
```diff
@@ -7,63 +7,56 @@ contributors:
...
 * [Getting started](./getting-started/index.md)
 * [Getting the Experience Platform SDKs](./getting-started/get-the-sdk.md)
 * [Frequently asked questions](./faq.md)
-* [Why use the Adobe Experience Platform Mobile SDKs](https://medium.com/adobetech/accelerate-your-mobile-application-development-with-adobe-experience-platform-mobile-sdk-and-launch-ed023536d611)
\ No newline at end of file
+* [Why use the Adobe Experience Platform Mobile SDKs](https://medium.com/adobetech/accelerate-your-mobile-application-development-with-adobe-experience-platform-mobile-sdk-and-launch-ed023536d611)
+
+## Get help
+
+If you have any questions about the SDK, please visit the [SDK community forum](https://experienceleaguecommunities.adobe.com/t5/adobe-experience-platform-sdks/ct-p/platform-sdk). If you need immediate assistance, please contact [Adobe Experience Cloud customer care](https://experienceleague.adobe.com/?support-solution=General#support).
```
Comment (emdobrin) [inline src/pages/documentation/index.md:62]: Let's put these two sentences on different lines.

File: src/pages/documentation/index.md [hunk 4]
```diff
@@ -7,63 +7,56 @@ contributors:
...
-- [Xamarin](#)
+- [Android](./current-sdk-versions.md#android)
+- [iOS - Swift](./current-sdk-versions.md#ios--swift)
+- [iOS - Objective-C](./current-sdk-versions.md#ios--objective-c)
+- [React Native (AEP 1.x)](./current-sdk-versions.md#react-native-aep-1x)
+- [React Native (ACP 2.x)](./current-sdk-versions.md#react-native-acp-2x)
+- [Flutter (AEP 1.x)](./current-sdk-versions.md#flutter-aep-1x)
+- [Flutter (ACP 2.x)](./current-sdk-versions.md#flutter-acp-2x)
+- [Cordova](./current-sdk-versions.md#cordova)
+- [WeChat Mini Programs](./platform-minis)
```
Comment (emdobrin) [inline src/pages/documentation/index.md:44]: ./platform-minis/index.md

**Comment by cacheung** [comment]
@calebk1 @emdobrin 
Per PM, we can remove WeChat platform-minis files in this repo.
We will keep it in gitBook.

**Comment by calebk1** [comment]
@cacheung If that's the case, you'll need to remove it from the gatsby-config.js file as well!

**Comment by cacheung** [comment]
> @cacheung If that's the case, you'll need to remove it from the gatsby-config.js file as well!

Thanks for the info.  Updated.

**Review by emdobrin — APPROVED** [review]

**Review by calebk1 — APPROVED** [review]
LGTM!

**Pull request merged**